File: app/src/main/java/net/hearnsoft/tcm/utils/Logs.java
package net.hearnsoft.tcm.utils;

import android.util.Log;
import net.hearnsoft.tcm.BuildConfig;


public class Logs {

    public static void i(String tag, String msg) {
        Log.i(tag, msg);
    }

    public static void d(String tag, String msg) {
        if (BuildConfig.DEBUG) {
            Log.d(tag,msg);
        }
    }

    public static void e(String tag, String msg) {
        if (BuildConfig.DEBUG) {
            Log.e(tag,msg);
        }
    }

}

File: app/src/main/java/net/hearnsoft/tcm/utils/Logs.kt
package net.hearnsoft.tcm.utils

import android.util.Log
import net.hearnsoft.tcm.BuildConfig

object Logs {
    @JvmStatic
    fun d(tag: String, message: String) {
        if (BuildConfig.DEBUG) {
            Log.d(tag, message)
        }
    }
    @JvmStatic
    fun e(tag: String, message: String) {
        if (BuildConfig.DEBUG) {
            Log.e(tag, message)
        }
    }
    @JvmStatic
    fun i(tag: String, message: String) {
        Log.i(tag, message)
    }
    @JvmStatic
    fun w(tag: String, message: String) {
        Log.w(tag, message)
    }

    @JvmStatic
    fun e(tag: String, message: String, throwable: Throwable) {
        if (BuildConfig.DEBUG) {
            Log.e(tag, message, throwable)
        }
    }
}